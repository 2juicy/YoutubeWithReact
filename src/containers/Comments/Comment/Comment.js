import React from "react";
import "./Comment.scss";
import { Image, Button } from "semantic-ui-react";
import Rating from "../../../components/Rating/Rating";

export default function Comment() {
  return (
    <div className="comment">
      <Image
        className="user-image"
        src="http://via.placeholder.com/48x48"
        circular
      />
      <div>
        <div className="user-name">User name</div>
        <span>Comment text</span>
        <div className="comment-actions">
          <Rating hideProgress likes={21} dislikes={1} />
          <Button size="mini" compact>
            REPLY
          </Button>
        </div>
      </div>
    </div>
  );
}
