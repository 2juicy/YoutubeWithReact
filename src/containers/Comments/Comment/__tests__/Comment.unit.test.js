import React from "react";
import { shallow } from "enzyme";
import Comment from "../Comment";

describe("Comment", () => {
  test("renders", () => {
    const wrapper = shallow(<Comment />);
    expect(wrapper).toMatchSnapShot();
  });
});
