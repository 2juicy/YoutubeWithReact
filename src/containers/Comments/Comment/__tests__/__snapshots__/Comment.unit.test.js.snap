// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment renders 1`] = `
<div
  className="comment"
>
  <Image
    as="img"
    circular={true}
    className="user-image"
    src="http://via.placeholder.com/48x48"
    ui={true}
  />
  <div>
    <div
      className="user-name"
    >
      User name
    </div>
    <span>
      Comment text
    </span>
    <div
      className="comment-actions"
    >
      <Rating
        dislikes={1}
        hideProgress={true}
        likes={21}
      />
      <Button
        as="button"
        compact={true}
        size="mini"
      >
        REPLY
      </Button>
    </div>
  </div>
</div>
`;
